framework/UPGRADE.md

```diff
@@ -20,6 +20,10 @@ ______________________
 * The signature of `yii\helpers\BaseArrayHelper::index()` was changed. The method has got an extra optional parameter
   `$groupBy`.
 
+* The signature of `yii\db\ColumnSchemaBuilder::__construct()` was changed. The method has got an extra optional parameter `$db`.
+
+* String types in the MySQL column schema map were upgraded to unicode storage types.
```

Comment:
UPGRADE.md must answer the question: `What must I do to upgrade from x.x.x to x.x.y`.
What must I do with this change? :)